contracts/ZapYCRV.vy

```diff
@@ -32,76 +35,36 @@ CRV: constant(address) =        0xD533a949740bb3306d119CC777fa900bA034cd52 # CRV
 YVBOOST: constant(address) =    0x9d409a0A012CFbA9B15F6D4B36Ac57A46966Ab9a # YVBOOST
 YCRV: constant(address) =       0xFCc5c47bE19d06BF83eB04298b026F81069ff65b # YCRV
 STYCRV: constant(address) =     0x27B5739e22ad9033bcBf192059122d163b60349D # ST-YCRV
-LPYCRV: constant(address) =     0xc97232527B62eFb0D8ed38CF3EA103A6CcA4037e # LP-YCRV
-POOL: constant(address) =       0x453D92C7d4263201C69aACfaf589Ed14202d83a4 # POOL
+LPYCRV_V1: constant(address) =  0xc97232527B62eFb0D8ed38CF3EA103A6CcA4037e # LP-YCRV Deprecated
+LPYCRV_V2: constant(address) =  0x6E9455D109202b426169F0d8f01A3332DAE160f3 # LP-YCRV
+POOL_V1: constant(address) =    0x453D92C7d4263201C69aACfaf589Ed14202d83a4 # OLD POOL
+POOL_V2: constant(address) =    0x99f5aCc8EC2Da2BC0771c32814EFF52b712de1E5 # NEW POOL
+POOL_OLD: constant(address) =   0x453D92C7d4263201C69aACfaf589Ed14202d83a4 # POOL
```

Comment:
`POOL_V1` and `POOL_OLD` are the same address...

Reply:
yeah just delete `POOL_OLD`, it's not used anywhere